mteb/benchmarks/benchmarks.py

```diff
@@ -734,7 +787,7 @@ def load_results(
 
 
 MTEB_INDIC = Benchmark(
-    name="MTEB(indic)",
+    name="MTEB(indic, beta)",
```

Comment:
Is there a reason, why indic shouldn't be capitalized? Do we have a unified scheme for naming benchmarks?

Reply:
That was my bad, should have been `Indic`

Reply:
fixed